test/rpc-server/logTiming/response.golden

```diff
@@ -0,0 +1 @@
+{"jsonrpc":"2.0","id":1,"result":{"reason":"stuck","depth":3,"state":{"term":{"format":"KORE","version":1,"term":{"tag":"App","name":"Lbl'-LT-'generatedTop'-GT-'","sorts":[],"args":[{"tag":"App","name":"Lbl'-LT-'k'-GT-'","sorts":[],"args":[{"tag":"App","name":"kseq","sorts":[],"args":[{"tag":"App","name":"inj","sorts":[{"tag":"SortApp","name":"SortState","args":[]},{"tag":"SortApp","name":"SortKItem","args":[]}],"args":[{"tag":"DV","sort":{"tag":"SortApp","name":"SortState","args":[]},"value":"f"}]},{"tag":"App","name":"dotk","sorts":[],"args":[]}]}]},{"tag":"App","name":"Lbl'-LT-'generatedCounter'-GT-'","sorts":[],"args":[{"tag":"DV","sort":{"tag":"SortApp","name":"SortInt","args":[]},"value":"0"}]}]}}}}}
```

Comment:
I don't see the log in the response, is this intended?

Reply:
Yes. This response isi for `logTiming: false`. If we have logs, we don't know what time will be shown (the python code just checks that it is > 0)

Reply:
given we have logic for when the timing logs are emitted, should we have one golden test which includes them?

Reply:
A bit difficult because we would need to replace the time value with a known constant (the comparison with the golden file is just textual). Of course doable with some regex-and-replacement in python but I did not think it would add much value.